app/features/settings/SettingsForm.js

```diff
@@ -97,6 +97,12 @@ export function SettingsForm({ minWk, maxWk, handleSubmit, submitting, submitSuc
           normalize={infoLevelNameToNum}
           note="Incorrect lessons are always high detail."
         />
+        <Field
+          name="repeatUntilCorrect"
+          label="Repeat item until correct:"
```

Comment:
```suggestion
          label="Repeat quiz question until correct:"
```

This should probably be clearer for users so they understand what `item` is here.